src/app/app.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="flash-message">
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/79994717-fb1f3300-84be-11ea-8cdf-698eb4ab270a.png)

Reply:
![image](https://user-images.githubusercontent.com/54113846/79994767-0a9e7c00-84bf-11ea-870a-f6732cbfa61b.png)

Reply:
Margins should be the same as on the mockup (Breakpoint >1200). Please, also check it on the other breakpoints

Reply:
![image](https://user-images.githubusercontent.com/54113846/79995873-70d7ce80-84c0-11ea-9e24-12af7232e2b3.png)

Reply:
![image](https://user-images.githubusercontent.com/54113846/79995895-79c8a000-84c0-11ea-8b6d-3df50b98d2f3.png)

Reply:
Please, fix paddings according to the mockup

Reply:
![image](https://user-images.githubusercontent.com/54113846/79996030-a2509a00-84c0-11ea-8c38-29123c0f7fde.png)

Reply:
Active item in the header should be green

Reply:
![image](https://user-images.githubusercontent.com/54113846/79996154-c57b4980-84c0-11ea-9ff8-867c0aa35fa3.png)

Reply:
![image](https://user-images.githubusercontent.com/54113846/79996778-7eda1f00-84c1-11ea-8459-9c5020d413d5.png)

Reply:
Please, check dimensions for hamburger menu and also paddings between lines.

Reply:
![image](https://user-images.githubusercontent.com/54113846/79997124-e1cbb600-84c1-11ea-9cd2-4b297a797169.png)

Reply:
Add 'border-radius' for every line

Reply:
fix